cms/static/js/views/utils/view_utils.js

```diff
@@ -5,7 +5,8 @@ define(["jquery", "underscore", "gettext", "js/views/feedback_notification", "js
     function ($, _, gettext, NotificationView, PromptView) {
         var toggleExpandCollapse, showLoadingIndicator, hideLoadingIndicator, confirmThenRunOperation,
             runOperationShowingMessage, disableElementWhileRunning, getScrollOffset, setScrollOffset,
-            setScrollTop, redirect, reload, hasChangedAttributes, deleteNotificationHandler;
+            setScrollTop, redirect, reload, hasChangedAttributes, deleteNotificationHandler,
```

Comment:
Actually, given the layout of this file, we should probably be declaring each of these vars on the same line where we assign them. Assigning a variable far away from its declaration (170 lines away here) often leads to bugs/typos/omissions.

Naturally, let's not change it for all of the other variables, but what about moving this declaration below:
```javascript
var keywordValidator = (function () {
    // ...
}());
```